### PR TITLE
Add Manual QA checklist to PR template

### PR DESCRIPTION
**Pull request by maastha**
## Description

Add a checklist for manual QA to PR template for this repository.

Link to any related issue(s): 

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Required Checklist:

- [X] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have run `make fmt` and formatted my code
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation (if appropriate)
- [ ] For CFN Resources: I have released by changes in the private registry and proved by change works in Atlas

### CODE REVIEW

**Review by Zuhairahmed — DISMISSED** [review]
This is a good one to have to also help customer experience too @maastha! LGTM

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -2,22 +2,35 @@
...
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
+  expected)
 - [ ] This change requires a documentation update
 
+## Manual QA performed:
+
+- [ ] cfn invoke for each of CRUDL/cfn test
+- [ ] Updated resource in ../example
+- [ ] Published to AWS private registry
+- [ ] Created and updated resource in AWS
```
Comment (andreaangiolillo) [inline .github/pull_request_template.md:20]:
```suggestion
- [ ] Use the template in [example](https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/examples) to create a stack in AWS
```

Comment (andreaangiolillo) [inline .github/pull_request_template.md:20]:
```suggestion
- [ ] Use the template in [example](https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/examples) to create a stack in AWS
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -2,22 +2,35 @@
...
 - [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
+  expected)
 - [ ] This change requires a documentation update
 
+## Manual QA performed:
+
+- [ ] cfn invoke for each of CRUDL/cfn test
+- [ ] Updated resource in ../example
```
Comment (andreaangiolillo) [inline .github/pull_request_template.md:18]:
```suggestion
- [ ] Updated resource in  [example](https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/examples)
```

File: .github/pull_request_template.md (hunk 1 above)
Comment (maastha) [reply .github/pull_request_template.md:20]: @andreaangiolillo do we not want to test updating a resource?

Comment (andreaangiolillo) [reply .github/pull_request_template.md:20]: yes, I missed it in my suggested changes

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -2,22 +2,35 @@
 
 Please include a summary of the fix/feature/change, including any relevant motivation and context.
 
-Link to any related issue(s): 
+Link to any related issue(s):
```
Comment (andreaangiolillo) [inline .github/pull_request_template.md:5]:
```suggestion
<!--
What MongoDB CLI issue does this PR address? (for example, #1234), remove this section if none.
-->

Closes #[issue number]
```

[the remark this answers is not in the document]
Comment (maastha) [reply .github/pull_request_template.md]: @andreaangiolillo Updated template, let me know if this looks good.
@Zuhairahmed @andreaangiolillo Do we need the 'Type of change' section?

[the remark this answers is not in the document]
Comment (andreaangiolillo) [reply .github/pull_request_template.md]: > Do we need the 'Type of change' section?

yes

**Review by andreaangiolillo — APPROVED** [review]
LGTM

**Pull request merged**